llama-index-integrations/llms/llama-index-llms-zhipuai/pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.llms.zhipuai"
+
+[tool.llamahub.class_authors]
+ZhipuAI = "llama-index"
```

Comment:
You should use your github username here instead of default "llama-index"

Reply:
ok, fix it